src/Standards/BestIt/Sniffs/Commenting/AbstractDocSniff.php

```diff
@@ -4,6 +4,7 @@
 
 namespace BestIt\Sniffs\Commenting;
 
+use BestIt\CodeSniffer\CodeError;
 use BestIt\CodeSniffer\CodeWarning;
```

Comment:
This seems to be obsolete now.

Reply:
processToken() should also throw an error now.

Reply:
> This seems to be obsolete now.

Not necessarily. This is the declaration for the usage in line 379 ... 

> processToken() should also throw an error now.

... but this could be obsolete, because CodeError extends CodeWarning. But good eye, i see your point. Will add this!